.github/workflows/release.yml

```diff
@@ -177,68 +155,3 @@ jobs:
                       }
               env:
                   SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-
-    report-release-success:
```

Comment:
I removec that success notification because the release is technically complete only once the bundle was published both to Apphub and to d2-ci. I became aware of this because there was a period of time where the bundle wasn't being pushed to d2-ci, which resulted in dhis2 core being released without the latest of our apps.

Reply:
After more consideration, I added the success message back in to the release workflow. And now publish-d2-ci will only report its own failure.